app/src/androidTest/java/com/arygm/quickfix/ui/search/SearchWorkerResultScreenTest.kt

```diff
@@ -185,17 +185,19 @@ class SearchWorkerResultScreenTest {
 
   @Test
   fun testProfileResultsAreDisplayed() {
+
     // Set the composable content
     composeTestRule.setContent {
       SearchWorkerResult(navigationActions, searchViewModel, accountViewModel)
     }
     // Scroll through the LazyColumn and verify each profile result is displayed
     val workerProfilesList = composeTestRule.onNodeWithTag("worker_profiles_list")
+    composeTestRule.onNodeWithTag("worker_profiles_list").printToLog("LazyColumn")
```

Comment:
Maybe remove this as it is not needed on the test 😅

Reply:
indeed haha